{{cookiecutter.repo_name}}/requirements/test.txt

```diff
@@ -1,12 +1,13 @@
 # Packages required for testing
 -r base.txt
 
-coverage
-django-dynamic-fixture
-django-nose
-mock
-nose-ignore-docstring
-pep8
-pylint
+coverage == 4.0.2
```

Comment:
Why are versions being set? I intentionally avoided hardcoding versions to prevent us from creating new projects with outdated dependencies. If we must set a version, use a range.